### PR TITLE
IPP HTML Receipts are missing fees and shipping

### LINKED ISSUE #3984: IPP HTML Receipts are missing fees and shipping
### Describe the bug

Receipts endpoint introduced [here](https://github.com/Automattic/woocommerce-payments/pull/3433) - `payments/readers/receipts/$payment_id` - is not showing fees and shipping items.

### To Reproduce

1. Create an order with fees and shipping items
2. Load the receipts from the endpoint
3. Notice the receipt `total` includes everything but rows for fees + shipping items are missing

### Screenshots

Screenshot 2022-03-18 at 14 11 28

### Expected behavior

HTML receipts should include all parts of the order - the same items which are listed on Order Detail on the web.
cc @mgascam

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #3984 

#### Changes proposed in this Pull Request
- Add fees and shipping information to receipts. 
- Return the receipt HTML on `html_content` key 

#### Testing instructions
 - Connect your local WC Pay client and server.
 - Create an order with Cash on Delivery as payment method. Please use the proxied url of your local site.
 - Start listening for webhooks in your local WC Pay server. You'll need this to refresh your order status after next step.
 - Use the WooCommerce Mobile App to pay the order. You can follow this great guide to set it up locally: pdjVC1-9M-p2
 - Once the payment is suceeded, you'll need to note the payment id reference i.e: you can find it in the order details
 - Using Postman (note that you'll need to set up Authentication as explained in this guide: PbTz5e-12J-p2) make a GET request to /wp-json/wc/v3/payments/readers/receipts/<payment_id>.
 - You should be able to check the HTML response within Postman. The HTML response should be on `html_content` key

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

File: changelog/fix-3984-add-shipping-fee [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+IPP HTML Receipts are missing fees and shipping
```
Comment (kalessil) [inline changelog/fix-3984-add-shipping-fee:4]: `In-Person Payments: receipts are missing fees and shipping`

File: includes/admin/class-wc-rest-payments-reader-controller.php [hunk 2]
```diff
@@ -296,7 +296,7 @@ public function generate_print_receipt( $request ) {
 			];
 
 			/* Generate receipt */
-			$response = $this->receipts_service->get_receipt_markup( $settings, $order, $charge );
+			$response = [ 'html_content' => $this->receipts_service->get_receipt_markup( $settings, $order, $charge ) ];
```
Comment (kalessil) [inline includes/admin/class-wc-rest-payments-reader-controller.php:299]: I'm not sure if we change the response structure here, can you share details behind this change, please?

Comment (kalessil) [reply includes/admin/class-wc-rest-payments-reader-controller.php:299]: @mgascam shared context for the change, hence ignore my previous question.

**Review by mgascam — APPROVED** [review]
LGTM! I did some tests and fees and shipping are showing correctly now, great job @dpaun1985

**Review by anu-rock — APPROVED** [review]
LGTM 🎉 (I did a code-only review). Thanks for picking this up and quickly fixing the issue.

A question: Is the `html_content` change related to #3901?

**Pull request merged**